app/controllers/recipes_controller.rb

```diff
@@ -0,0 +1,6 @@
+class RecipesController < ApplicationController
+  def show
+    @recipe = Recipe.find(params[:id])
+    @ingredients = @recipe.ingredients
```

Comment:
This is fine; you can also access `@recipes.ingredients` from the view itself, which will help keep your controller methods "skinny".